shared/my-sites/themes/jetpack-manage-disabled-message.jsx

```diff
@@ -21,10 +23,37 @@ export default React.createClass( {
 		} ).isRequired
 	},
 
-	clickOnActivate: function() {
+	clickOnActivate() {
 		analytics.ga.recordEvent( 'Jetpack', 'Activate manage', 'Site', this.props.site ? this.props.site.ID : null );
 	},
 
+	renderMockThemes() {
+		const exampleThemesData = [
+			{ name: 'Boardwalk', slug: 'boardwalk' },
+			{ name: 'Cubic', slug: 'cubic' },
+			{ name: 'Edin', slug: 'edin' },
+			{ name: 'Cols', slug: 'cols' },
```

Comment:
This theme doesn't exits in the .org theme repo so I would replace it with something that is.